packages/xo-web/src/common/form/select.js

```diff
@@ -33,7 +31,7 @@ export default class Select extends React.PureComponent {
   static propTypes = {
     autoSelectSingleOption: PropTypes.bool, // default to props.required
     maxHeight: PropTypes.number,
-    options: PropTypes.array.isRequired, // cannot be an object
+    options: PropTypes.array, // cannot be an object
```

Comment:
Is `null` ok with this? Should we specifically say that `null` is accepted?

Reply:
Yes, It's not required. 
So, It can be `array` or `null` or `undefined`.